velox/docs/functions/spark/config.rst

```diff
@@ -0,0 +1,21 @@
+========================
+SparkRegistration Configuration properties
+========================
+
+Generic Configuration
```

Comment:
`allowPrecisionLoss` only applies to several functions so it is not a generic configuration. I feel this document is not needed as the description in `struct SparkRegistrationConfig`  is clear enough.